content/developer/reference/addons/views.rst

```diff
@@ -1131,48 +1131,51 @@ The graph view is used to visualize aggregations over a number of records or
 record groups. Its root element is ``<graph>`` which can take the following
 attributes:
 
-``type``
+``type`` (optional)
   one of ``bar`` (default), ``pie`` and ``line``, the type of graph to use
-``stacked``
-  only used for ``bar`` charts. If present and set to ``True``, stacks bars
-  within a group
-``disable_linking``
-  set to ``True`` to prevent from redirecting clicks on graph to list view
-``order``
+
+``stacked`` (optional)
+  only used for ``bar`` charts. Set to ``0`` to prevent the bars within a group
+  to be stacked initially.
+
+``disable_linking`` (optional)
+  set to ``1`` to prevent from redirecting clicks on graph to list view
+
+``order`` (optional)
   if set, x-axis values will be sorted by default according their measure with
   respect to the given order (``asc`` or ``desc``). Only used for ``bar`` and
   ``pie`` charts.
 
+``string`` (optional)
+  string displayed in the breadcrumbs when redirecting to list view.
+
 The only allowed element within a graph view is ``field`` which can have the
 following attributes:
 
-``name`` (required)
+``name`` (mandatory)
   the name of a field to use in the view. If used for grouping (rather
   than aggregating)
 
-``title`` (optional)
-  string displayed on the top of the graph.
-
 ``invisible`` (optional)
   if true, the field will not appear either in the active measures nor in the
   selectable measures.
 
-``type``
-  indicates whether the field should be used as a grouping criteria or as an
-  aggregated value within a group. Possible values are:
+``type`` (optional)
+  if set to ``measure``, the field will be used as an aggregated value within a
+  group instead of a grouping criteria. Remark: only works for the last field
+  with that attribute but it is useful for other fields with string attribute
+  (see below).
```

Comment:
```suggestion
  group instead of a grouping criteria. It only works for the last field
  with that attribute but it is useful for other fields with string attribute
  (see below).
```